lib/actions-oclif/os/configure.ts

```diff
@@ -213,14 +225,36 @@ export default class OsConfigureCmd extends Command {
 
 		console.info('Configuring operating system image');
 
+		const image = params.image;
 		await helpers.osProgressHandler(
 			await devInit.configure(
-				params.image,
+				image,
 				deviceTypeManifest,
 				configJson || {},
 				answers,
 			),
 		);
+
+		if (options["system-connection"]) {
+			const files = await Bluebird.map(options["system-connection"], async filePath => {
+				const content = await fs.promises.readFile(filePath, 'utf8');
+				const name = path.basename(filePath);
+
+				return {
+					name,
+					content
+				};
+			});
+
+			await Bluebird.each(files, async ({ name, content }) => {
```

Comment:
`Bluebird.each` writes files one a time, rather than in parallel. I guess it's intentional.

Reply:
I didn't know if parallel might have consequences with it being a disk image file, so I played it safe in the knowledge that the files *should* be small text files.